locale/da/texts.po

```diff
@@ -668,7 +668,6 @@ msgid "Disable biometry"
 msgstr "Deaktiver biometri"
 
 #: src/screens/Security.js:121
-#, fuzzy
```

Comment:
These translations looks correct (according to chatgpt), but fuzzy translations are generated automatically, and are usually wrong